cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -41,7 +41,7 @@ the 'cdap' user installed by the parcel.
 Prerequisites
 =======================================
 
-#. Node.js (version 0.8.16 through 0.10.37) must be installed on the node(s) where the Web-App
+#. Node.js (version 0.8.16 through 0.10.37) must be installed on the node(s) where the UI
```

Comment:
Check this with the updated supported versions for the UI, versus the old webapp. Pretty sure it supports other versions of node with the new UI.